Samples/MedicationsTracker/MedicationsTracker/MedicationBuilder.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MedicationBuilder.swift
+//  MedicationTracker
+//
+//  Created by Kayla Davis on 5/31/17.
+//  Copyright © 2017 Kayla Davis. All rights reserved.
+//
+
+import Foundation
+
+class MedicationBuilder {
+    var thing: MHVThing?
+    var med: MHVMedication?
+    
+    func begin(mhvThing: MHVThing) -> MedicationBuilder{
+        thing = mhvThing
+        med = mhvThing.medication()
+        return self
+    }
+    
+    func setName(name: String) -> MedicationBuilder {
```

Comment:
See earlier about returning Bool if these succeed or fail.

In ObjectiveC, set is a special prefix for a setter and those typically don't return a value, to be safe, maybe updateName() instead